mentorship ios/ViewModels/LoginViewModel.swift

```diff
@@ -30,5 +30,11 @@ class LoginViewModel: ObservableObject {
     func attemptAppleLogin() {
         appleSignInCoordinator.handleAuthorizationAppleIDButtonPress()
     }
+
+    //reset tthe login data when the view is going to disappear
+    //fix for:https://github.com/anitab-org/mentorship-ios/issues/125
```

Comment:
Line 34: typo (also, can be made more explicit in intention, example: `used to reset the login data on a successful login`)
Line 35: not really required